### PR TITLE
feat(AV-4419): update us verifications response body

### PR DESCRIPTION
**Pull request by amaan-lob**
## Description

Updating US Verifications response to include new fields Cycle O fields such as po_box_only_flag, dpv_no_secure_location, and dpv_door_not_accessible. 

## Story

https://lobsters.atlassian.net/browse/AV-4395
https://lobsters.atlassian.net/browse/AV-4419

## Related PR's

https://github.com/lob/sdks_openapi_gen/pull/334

### CODE REVIEW

**Review by ajorczak-lob — APPROVED** [review]

File: docs/DeliverabilityAnalysis.md [hunk 1]
```diff
@@ -24,6 +30,12 @@ instance = Lob::DeliverabilityAnalysis.new(
   dpv_cmra: null,
   dpv_vacant: null,
   dpv_active: null,
+  dpv_inactive_reason: null,
```
Comment (ajorczak-lob) [inline docs/DeliverabilityAnalysis.md:33]: Need to update nulls to nil here. `null` in Ruby is `nil`
```suggestion
  dpv_inactive_reason: nil,
```

**Review by ajorczak-lob — CHANGES_REQUESTED** [review]
Just some doc updates to change null from nil that was autogened

File: docs/DeliverabilityAnalysis.md (hunk 1 above)
Comment (amaan-lob) [reply docs/DeliverabilityAnalysis.md]: Updated the docs I updated here, but I'll set a reminder to update this in the sdk generator repo

Comment (amaan-lob) [reply docs/DeliverabilityAnalysis.md]: Fixed the docs I updated here, but I'll set a reminder to update this in the sdk generator repo

**Review by mrkaspa — APPROVED** [review]

**Review by ajorczak-lob — APPROVED** [review]

**Pull request merged**